packages/flutter_tools/lib/src/persistent_tool_state.dart

```diff
@@ -52,6 +52,8 @@ abstract class PersistentToolState {
 
   /// Whether this client was already determined to be or not be a bot.
   bool isRunningOnBot;
+
+  DateTime lastDevToolsActivationTime;
```

Comment:
Nit, doc comment on what this is used for and link to the devtools launcher class

Reply:
done